index.html

```diff
@@ -0,0 +1,971 @@
+              </g>
+            </svg>
+        </div>
+        <nav class="header-bottom" id="navbottom">
+        <ul class="page-nav">
+            <li><a>U.S.</a></li>
+            <li><a>World</a></li>
+            <li><a>Bisiness</a></li>
+            <li><a>Tech & Science</a></li>
+            <li><a>Culture</a></li>
+            <li><a>Newsgeek</a></li>
+            <li><a>Sports</a></li>
+            <li><a>Health</a></li>
+            <li><a>Opinion</a></li>
+            <li><a>Vantage</a></li>
+            <li><a id="search">Search</a><i class="fas fa-search"></i></li>
+        </ul>
+    </nav>
+    <div class="content">
+        <div class="featured">
+            <div class="row">
+                <div class="col-md-4 col-lg order-2 order-md-1">
+                    <div class="featured">
+                        <div class="block-title">FEATURED STORIES</div>
+                        <article>
+                            <div class="wrap">
+                                <a><img src="images/camarillo-california-wildfire-climate-change-getty.jpg" class="img-fluid" alt="Responsive image"><span class="pic-on">TECH & SCIENCE</span></a>
+                            </div>
+                            <div class="article-info">
+                                <h3>What Do Evangelical Christians Really Think About Climate Change?</h3>
+                                <div class="summary"> Newsweek spoke to two academics who immersed themselves in evangelical communities to understand how a quarter of the U.S. population came to reject the idea that climate change is a man-made problem.</div>
+                            </div>
+                        </article>
+                        <article>
+                            <div class="wrap">
+                                <a><img src="images/john-bolton.jpg" class="img-fluid" alt="Responsive image"><span class="pic-on">Politics</span></a>
+                            </div>
+                            <div class="article-info">
+                                <h3>John Bolton Slams Trump's Foreign Policy at Private Lunch: Report</h3>
+                                <div class="summary">The former national security adviser also reportedly asserted that President Donald Trump's negotiations with Iran and North Korea are "doomed to failure."</div>
+                            </div>
+                            <!--<div class="related">
```

Comment:
remove this comment

Reply:
Hey bro, what's your official name? I can't find you in the TSEs list.

Reply:
Martin Afani